setup.cfg

```diff
@@ -1,3 +1,53 @@
 [metadata]
-description-file = README.md
-license-file = LICENSE.txt
+name = colorcet
+version = attr: param.version.get_setupcfg_version
+long_description = file: README.md
+long_description_type = text/markdown
+license_file = LICENSE.txt
+classifiers =
+    License :: Other/Proprietary License
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3.6
+    Development Status :: 5 - Production/Stable
+author = James A. Bednar
+author_email = jbednar@continuum.io
+maintainer = James A. Bednar
+maintainer_email = jbednar@continuum.io
+url = https://colorcet.pyviz.org
+project_urls =
+    Bug Tracker = http://github.com/bokeh/colorcet/issues
+    Documentation = https://colorcet.pyviz.org
+    Source Code = http://github.com/bokeh/colorcet
+
+[options]
+include_package_data = True
+packages = find:
+python_requires = >=2.7
+install_requires =
+    param >=1.7.0
+
+[options.extras_require]
+tests =
+    flake8
+    nbsmoke >=0.2.6
+    pytest >=2.8.5
+
+doc =
+    nbsite
+    sphinx_ioam_theme
+
+examples =
+    pyct[cmd]
+    numpy
+    holoviews
+
+[options.entry_points]
+console_scripts =
+    colorcet = colorcet.__main__:main
+
+[wheel]
+universal = 1
```

Comment:
I think this didn't make it in (@jsignell). Without it, anyone building a wheel has to specify "universal" on the command line to get a universal wheel (I think).